src/Lesson_7/Arrays_1/ArraysPartOne.java

```diff
@@ -1,4 +1,93 @@
 package Lesson_7.Arrays_1;
 
 public class ArraysPartOne {
+
+    public static int[] intArray = new int[]{1, 2, 3};
```

Comment:
```suggestion
    public static int[] intArray = new int[3];
    static {
        for (int i = 0; i < intArray.length; i++) {
            intArray = i + 1;
        }
    }
```
Вот так нужно было сделать по заданию, а не через создание массива сразу с элементами, потому что такая форма записи равносильна просто `{1, 2, 3}`

Reply:
Не считал из задания...